tests/capi/unittest_capi_inference_nnfw_runtime.cc

```diff
@@ -324,97 +374,59 @@ TEST (nnstreamer_nnfw_mlapi, invoke_single_02_n)
   EXPECT_EQ (data_size, sizeof (float) * 16);
   data[0] = 10.0;
 
-  status = ml_single_invoke (single, input, &output);
+  status = ml_single_invoke (single_h, input, &output);
   EXPECT_EQ (status, ML_ERROR_INVALID_PARAMETER);
-
-  ml_tensors_data_destroy (output);
-  ml_tensors_data_destroy (input);
-
-  status = ml_single_close (single);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  g_free (test_model);
-  ml_tensors_info_destroy (in_info);
-  ml_tensors_info_destroy (out_info);
-  ml_tensors_info_destroy (in_res);
 }
 
 /**
- * @brief Callback for tensor sink signal.
+ * @brief Test nnfw subplugin with successful invoke (pipeline, ML-API)
  */
-static void
-new_data_cb (const ml_tensors_data_h data, const ml_tensors_info_h info, void *user_data)
+TEST_F (MLAPIInferenceNNFW, invoke_pipeline_00)
 {
   int status;
-  float *data_ptr;
+  float *data;
   size_t data_size;
-  int *checks = (int *)user_data;
-
-  status = ml_tensors_data_get_tensor_data (data, 0, (void **)&data_ptr, &data_size);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-  EXPECT_FLOAT_EQ (*data_ptr, 12.0);
-
-  g_mutex_lock (&g_test_mutex);
-  *checks = *checks + 1;
-  g_mutex_unlock (&g_test_mutex);
-}
-
-/**
- * @brief Test nnfw subplugin with successful invoke (pipeline, ML-API)
- */
-TEST (nnstreamer_nnfw_mlapi, invoke_pipeline_00)
-{
-  gchar *pipeline, *test_model;
-  ml_pipeline_h handle;
   ml_pipeline_src_h src_handle;
   ml_pipeline_sink_h sink_handle;
-  ml_tensor_dimension in_dim;
-  ml_tensors_info_h info;
+  // g_autofree guint *sink_called_cnt = nullptr;
```

Comment:
Unnecessary line.

Reply:
My fault! I will remove it in the next version.